src/include/common/utils.h

```diff
@@ -48,23 +49,11 @@ class StringUtils {
     }
 
     static bool CharacterIsDigit(char c) { return c >= '0' && c <= '9'; }
-    template<typename... Args>
```

Comment:
My general comment here is to still have our own StringUtils::string_format but internally we call fmt::format. This makes sure one day if we want to reimplement our string_format, we can quickly bring things back.

Reply:
done